knowledge-base/kb-gridview-style-aggregate-results-in-group-header.md

```diff
@@ -0,0 +1,358 @@
+---
+title: Style aggregate results displayed in the GroupHeaderRow
```

Comment:
Usually, we use capitals for the first letter of the words here. Similar to the titles of the articles in the help docs. Anyway, I don't have a strong argument why we can't use this casing here. The only concern is the consistency between the titles.

Can we discuss this further?